lib/internal/modules/helpers.js

```diff
@@ -320,6 +329,28 @@ function normalizeReferrerURL(referrerName) {
 }
 
 
+let esmSyntaxErrorMessagesSet; // Declared lazily in shouldRetryAsESM
+let throwsOnlyInCommonJSErrorMessagesSet; // Declared lazily in shouldRetryAsESM
+/**
+ * After an attempt to parse a module as CommonJS throws an error, should we try again as ESM?
+ * @param {string} errorMessage The string message thrown by V8 when attempting to parse as CommonJS
+ * @param {string} source Module contents
```

Comment:
returns is missing from this jsdoc

Reply:
`@returns` should never be added to JSDoc because it overrides what TypeScript infers as the return type from the function. It’s a way of casting the return type.

Reply:
TIL